docker-cloud/builds/automated-build.md

```diff
@@ -105,6 +105,17 @@ run an initial test.
     Docker Cloud on every push. Only pushes to branches that are listed as the
     source for one or more tags will trigger a build.
 
+### Environment variables for builds
+
+When you configure automated builds in Docker Cloud you can specify additional Environment Variables. The variables you add on the **Configure build settings** screen are used during the build process _only_.
+
+You can also declare environment variables in `hooks` files, however anything
+declared in these files can be read by anyone with read access to the code
+repository. If you use build environment variables to store access tokens or
+other information that should remain secret, create them on the build settings
+screen. The variables created here are only accessible to users who have `admin`
+access to the Docker Cloud repository.
```

Comment:
If it is not in other doc, I would say that env variables are only accessible from hooks, and I would add the example about how to pass these variables as build args of the dockerfile.